schemas/mss.xml

```diff
@@ -3752,18 +4093,70 @@ locus is used to point to values of these elements in the teiHeader.
                 
                 <!-- make the foliation element a member of att.datable (this is to enable identification of medieval foliation, if desired -->
                 <elementSpec ident="foliation" module="msdescription" mode="change">
+                    <!--Information about the systems used to number the folios or pages in the manuscript.-->
+                    
+                    <!--ADD @type, e.g. “arabic”, “hebrew” to make this feature machine readable?-->
                     <classes mode="change">
                         <memberOf key="att.datable"/>
                     </classes>
                 </elementSpec>
 
                 <!-- add new attributes "topLine" and "rulingMedium" to layout
                         make layout part of att.typed  -->
+                
+                <!--ADD new attribute "rulingPattern", 
+                values: 
+                -	Framework only
+                -	Grid of single vertical and horizontal lines
+                -	Presence of additional lines
+                -	Variable instead of repeating “Presence of additional lines”-->
+                
+                <!--Do we need an attribute to establish whether there is ruling? In my view, the 
+                presence of the "ruling" attribute is sufficient.-->
+                
+                <!--ADD the "prickingMedium" attribute (values Flat blade, Awl, Compass) to "layout" -->
+                
+                <!--ADD the "prickingPattern" attribute (horizontal, vertical, horizontal & vertical, additional) to "layout" -->
+                
+                <!--ADD attribute (or element?) "justification"
+                values:
+                -	Left-hand margin is not even
+                -	Space fillers
+                -	Extended letters
+                -	Reduced letters
+                -	Broken letters
+                -	Anticipation
+                -	Abbreviation
+                -	Change of script register
+                -	Writing above the line horizontal
+                -	Writing above the line oblique
+                -	Writing below the line
+                -	Vertical writing
+                -	Change of orientation of letter components
+                -	Other (free-text box)	-->
+                
+                <!--ADD attribute (or element?) "textLayout"
+                values: 
+                -	No graphic marks of text navigation
+                -	Spaces between paragraphs
+                -	Paragraph marks and textual dividers (free-text box)
+                -	Rubric
+                -	Numbering of sections
+                -	Enlarged titles and incipits
+                -	Explicits
+                -	Colours
+                -	Change of script register
+                -	Change of script type
+                -	Decoration
+                -	Other (free-text box)-->
+                
+                
```

Comment:
These layout features might be worth discussing with the TEI-List. If there aren't yet suitable mechanisms for these they might consider adding some.